Fix "try it out" page so documented commands produce documented output

**Describe the bug**
Trying out Athens https://docs.gomods.io/try-out/ and it looks like `proxy` looks for `../../config.dev.toml` so it needs to be run from `$GOPATH/src/github.com/gomods/athens/`

**Error Message**
```
$ go get -u github.com/gomods/athens/cmd/proxy
$ proxy &
[1] 18689
$ 2018/10/13 17:23:43 open ../../config.dev.toml: no such file or directory
[1]+  Exit 1                  proxy
```

**To Reproduce**
1. `cd ~`
2. `go get -u github.com/gomods/athens/cmd/proxy`
3. `proxy &`

**Expected behavior**
As here: https://docs.gomods.io/try-out/
```
$ go get -u github.com/gomods/athens/cmd/proxy
# the source is downloaded to GOPATH/src/github.com/gomods/athens/
$ proxy &
[1] 25243
INFO[0000] Starting application at 127.0.0.1:3000
```

**Environment (please complete the following information):**
 - OS: macOSX 10.14.1 Beta
 - Go version : go version go1.11.1 darwin/amd64
 - Buffalo Version : ?
 - Olympus or proxy version : master 4b6da424807b9baabe7239bd709818046ce77899

**Additional context**
Would it be better to change proxy so it could run from anywhere with sensible defaults, or to add this line to the docs (between `go get` and `proxy &`):
```
$ cd $GOPATH/src/github.com/gomods/athens/cmd/proxy
```

Or both?

I can PR a fix for the docs and I could look into running the proxy without a config file if others think that's a worthwhile idea.

hmm, seems like there are some problems running the tests outside GOPATH:

```
$ GO111MODULE=on make verify test-unit test-e2e
./scripts/check_gofmt.sh
./scripts/check_golint.sh
cannot find package "_/Users/vout/athens/cmd/olympus" in any of:
	/usr/local/go/src/_/Users/vout/athens/cmd/olympus (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/cmd/olympus (from $GOPATH)
cannot find package "_/Users/vout/athens/cmd/olympus/actions" in any of:
	/usr/local/go/src/_/Users/vout/athens/cmd/olympus/actions (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/cmd/olympus/actions (from $GOPATH)
cannot find package "_/Users/vout/athens/cmd/olympus/grifts" in any of:
	/usr/local/go/src/_/Users/vout/athens/cmd/olympus/grifts (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/cmd/olympus/grifts (from $GOPATH)
cannot find package "_/Users/vout/athens/cmd/proxy" in any of:
	/usr/local/go/src/_/Users/vout/athens/cmd/proxy (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/cmd/proxy (from $GOPATH)
cannot find package "_/Users/vout/athens/cmd/proxy/actions" in any of:
	/usr/local/go/src/_/Users/vout/athens/cmd/proxy/actions (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/cmd/proxy/actions (from $GOPATH)
cannot find package "_/Users/vout/athens/cmd/proxy/grifts" in any of:
	/usr/local/go/src/_/Users/vout/athens/cmd/proxy/grifts (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/cmd/proxy/grifts (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/config" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/config (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/config (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/download" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/download (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/download (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/download/addons" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/download/addons (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/download/addons (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/errors" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/errors (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/errors (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/eventlog" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/eventlog (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/eventlog (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/eventlog/disposable" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/eventlog/disposable (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/eventlog/disposable (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/eventlog/fs" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/eventlog/fs (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/eventlog/fs (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/eventlog/mongo" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/eventlog/mongo (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/eventlog/mongo (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/eventlog/olympus" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/eventlog/olympus (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/eventlog/olympus (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/log" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/log (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/log (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/middleware" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/middleware (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/middleware (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/module" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/module (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/module (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/observ" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/observ (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/observ (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/paths" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/paths (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/paths (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/payloads" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/payloads (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/payloads (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/stash" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/stash (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/stash (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/storage" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/storage (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/storage (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/storage/azurecdn" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/storage/azurecdn (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/storage/azurecdn (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/storage/compliance" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/storage/compliance (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/storage/compliance (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/storage/fs" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/storage/fs (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/storage/fs (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/storage/gcp" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/storage/gcp (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/storage/gcp (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/storage/mem" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/storage/mem (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/storage/mem (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/storage/minio" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/storage/minio (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/storage/minio (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/storage/module" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/storage/module (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/storage/module (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/storage/mongo" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/storage/mongo (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/storage/mongo (from $GOPATH)
cannot find package "_/Users/vout/athens/pkg/storage/s3" in any of:
	/usr/local/go/src/_/Users/vout/athens/pkg/storage/s3 (from $GOROOT)
	/Users/vout/go/src/_/Users/vout/athens/pkg/storage/s3 (from $GOPATH)
# ./scripts/check_deps.sh TODO: comment back when we stop using vendor.
./scripts/check_conflicts.sh
./scripts/test_unit.sh
+ go test -mod=vendor -race -coverprofile cover.out -covermode atomic ./...
?   	github.com/gomods/athens/cmd/olympus	[no test files]
ok  	github.com/gomods/athens/cmd/olympus/actions	1.207s	coverage: 0.5% of statements
?   	github.com/gomods/athens/cmd/olympus/grifts	[no test files]
?   	github.com/gomods/athens/cmd/proxy	[no test files]
ok  	github.com/gomods/athens/cmd/proxy/actions	1.196s	coverage: 36.5% of statements
?   	github.com/gomods/athens/cmd/proxy/grifts	[no test files]
ok  	github.com/gomods/athens/pkg/config	1.052s	coverage: 61.6% of statements
ok  	github.com/gomods/athens/pkg/download	8.536s	coverage: 44.6% of statements
ok  	github.com/gomods/athens/pkg/download/addons	1.173s	coverage: 76.6% of statements
ok  	github.com/gomods/athens/pkg/errors	1.059s	coverage: 72.1% of statements
ok  	github.com/gomods/athens/pkg/eventlog	1.046s	coverage: 73.8% of statements
?   	github.com/gomods/athens/pkg/eventlog/disposable	[no test files]
ok  	github.com/gomods/athens/pkg/eventlog/fs	1.044s	coverage: 80.8% of statements
ok  	github.com/gomods/athens/pkg/eventlog/mongo	1.145s	coverage: 69.0% of statements
?   	github.com/gomods/athens/pkg/eventlog/olympus	[no test files]
ok  	github.com/gomods/athens/pkg/log	1.089s	coverage: 17.5% of statements
ok  	github.com/gomods/athens/pkg/middleware	1.202s	coverage: 89.1% of statements
ok  	github.com/gomods/athens/pkg/module	2.631s	coverage: 51.6% of statements
?   	github.com/gomods/athens/pkg/observ	[no test files]
?   	github.com/gomods/athens/pkg/paths	[no test files]
?   	github.com/gomods/athens/pkg/payloads	[no test files]
ok  	github.com/gomods/athens/pkg/stash	1.352s	coverage: 65.2% of statements
?   	github.com/gomods/athens/pkg/storage	[no test files]
?   	github.com/gomods/athens/pkg/storage/azurecdn	[no test files]
?   	github.com/gomods/athens/pkg/storage/compliance	[no test files]
ok  	github.com/gomods/athens/pkg/storage/fs	1.138s	coverage: 88.1% of statements
ok  	github.com/gomods/athens/pkg/storage/gcp	1.136s	coverage: 63.6% of statements
?   	github.com/gomods/athens/pkg/storage/mem	[no test files]
ok  	github.com/gomods/athens/pkg/storage/minio	1.119s	coverage: 0.0% of statements
ok  	github.com/gomods/athens/pkg/storage/module	3.141s	coverage: 96.3% of statements
ok  	github.com/gomods/athens/pkg/storage/mongo	1.111s	coverage: 0.0% of statements
--- FAIL: TestBackend (0.30s)
    <autogenerated>:1: 
        	Error Trace:	tests.go:19
        	            				s3_test.go:15
        	Error:      	Received unexpected error:
        	            	RequestError: send request failed
        	            	caused by: Get http://127.0.0.1:9001/gomods: dial tcp 127.0.0.1:9001: connect: connection refused
        	Test:       	TestBackend
        	Messages:   	clearing backend failed
FAIL
coverage: 8.0% of statements
FAIL	github.com/gomods/athens/pkg/storage/s3	0.397s
make: *** [test-unit] Error 1
```

--- a/docs/content/try-out.md
+++ b/docs/content/try-out.md
@@ -10,8 +10,17 @@ To quickly see Athens in action, follow these steps:
 
 First, make sure you have [Go 1.11 installed](https://gophersource.com/setup/),
 that GOPATH/bin is on your path, and that you have enabled the [Go
-Modules](https://github.com/golang/go/wiki/Modules) feature by exporting
-GO111MODULE=on.
+Modules](https://github.com/golang/go/wiki/Modules) feature.
+
+**Bash**
+```bash
+export GO111MODULE=on
+```
+
+**PowerShell**
+```powershell
+$env:GO111MODULE = "on"
+```
 
 Next, use git and Go to install and run the Athens proxy in a background process.
 

--- a/docs/content/try-out.md
+++ b/docs/content/try-out.md
@@ -8,33 +8,34 @@ menu: shortcuts
 
 To quickly see Athens in action, follow these steps:
 
-First, make sure you have [Go 1.11 installed](https://gophersource.com/setup/) and that GOPATH/bin is on your path.
+First, make sure you have [Go 1.11 installed](https://gophersource.com/setup/),
+that GOPATH/bin is on your path, and that you have enabled the [Go
+Modules](https://github.com/golang/go/wiki/Modules) feature by exporting
+GO111MODULE=on.
 
-Next, use Go to install and run the Athens proxy in a background process
+Next, use git and Go to install and run the Athens proxy in a background process.
 
 ```console
-$ go get -u github.com/gomods/athens/cmd/proxy
-# the source is downloaded to GOPATH/src/github.com/gomods/athens/
+$ git clone https://github.com/gomods/athens
+$ cd athens/cmd/proxy
+$ go install
 $ proxy &
-[1] 25243
-INFO[0000] Starting application at 127.0.0.1:3000
+[1] 37186
+INFO[0000] Exporter not specified. Traces won't be exported
+INFO[0000] Starting application at http://127.0.0.1:3000
 ```
 
-Next, you will need to enable the [Go Modules](https://github.com/golang/go/wiki/Modules)
-feature and configure Go to use the proxy!
+Next, you will need to configure Go to use the proxy!
 
 **Bash**
 ```bash
-export GO111MODULE=on
 export GOPROXY=http://127.0.0.1:3000
 ```
 
 **PowerShell**
 ```powershell
-$env:GO111MODULE = "on"
 $env:GOPROXY = "http://127.0.0.1:3000"
-``@ma`
-
+```
 
 Now, when you build and run this example application, **go** will fetch dependencies via Athens!
 

